API_doc/json_recipes/setting_update.json

```diff
@@ -1,6 +1,8 @@
 {
   "REQUEST_NAME": "SETTING_UPDATE",
   "CONFIG_ID": 42,
-  "SETTING_NAME": "foo",
-  "SETTING_VALUE": "bar"
+  "SETTINGS_TO_UPDATE": {
```

Comment:
Is this supposed to be a list? A json array might make more sens in that case.

Reply:
no this is an object

Reply:
I guess since it's key-value pairs, that makes sens.